src/generators/css-filter.ts

```diff
@@ -5,11 +5,24 @@ import {parseArray, formatArray} from '../utils/text';
 import {compareURLPatterns, isURLInList} from '../utils/url';
 import {createTextStyle} from './text-style';
 import {FilterConfig, InversionFix} from '../definitions';
+import {compareChromeVersions, getChromeVersion, isChromiumBased} from '../utils/platform';
 
 export enum FilterMode {
     light = 0,
     dark = 1
 }
+/**
+* Bug report:  https://bugs.chromium.org/p/chromium/issues/detail?id=501582
+* Patch: https://chromium-review.googlesource.com/c/chromium/src/+/1979258
+*/
+export function hasChromiumIssue501582() { 
+    const chromeVersion = getChromeVersion();
+    return Boolean(
+        chromeVersion &&
+        isChromiumBased() &&
```

Comment:
if `chromeVersion` is null, then there is no need to call this function.

Reply:
This?

Reply:
Check line 81